packages/sdks/mitosis.config.cjs

```diff
@@ -1180,11 +1180,18 @@ module.exports = {
           code: {
             post: (code, json) => {
               if (json.name === 'EnableEditor') {
+                code = code.replace('const elementRef = useSignal<Element>();', 'const elementRef = useSignal<Element>();\nconst hasExecuted = useSignal(false);');
```

Comment:
This is going to be a very brittle plugin to maintain. I would add the `hasExecuted` state to `enable-editor.lite.tsx`. It won't negatively impact the other SDKs to have this one extra state in there.

if we want, we can use `useTarget()` so that we only get and set this state for Qwik since it's unnecessary for the others